### DISCUSSION TITLE
[Weekly Curation Report]: July 31 - Aug 4

### DISCUSSION THREAD

**Discussion by AlexanderPico** [wikipathways/wikipathways-help#55] [curation-reports]
### Recent changes

Recent Changes: 22 pathways
- lots of great edits by Freddie -- thanks!
- updated approved tags that were lagging on about half of them
- manually fixed characters in bibliography of WP5380 and WP5345

### Under construction

Under Construction: 5 untouched in past 30 days
- approved 2
- left others authored by curators of the week

### New pathways

New pathways: 2 already approved

### Deleted Pathways

Deleted pathways: 1 tagged over 30 days ago

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks @AlexanderPico !

Here's a link for all Under Construction pathways, the highlighted pathways are those that have not been edited in the last 30 days: https://classic.wikipathways.org/index.php?title=Special:CurationTags&showPathwaysFor=Curation:UnderConstruction

**Reply by AlexanderPico** [reply]
Right. Those are the ones I reviewed.

**Reply by khanspers** [reply]
Sorry, my comment was unclear. I added the link for quick access for the rest of the Curators team to review :smile:

**Reply by AlexanderPico** [reply]
Oh, good idea!
